Ports/m4/package.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash ../.port_include.sh
+port=m4
+version=1.4.9
+useconfigure=true
+files="http://ftp.gnu.org/gnu/m4/m4-1.4.9.tar.gz m4-1.4.9.tar.gz"
+
+configure() {
+    run ./configure --host=386bsd
```

Comment:
Why are we using `--host=386bsd` here?

Reply:
Just override ``--host=i686-pc-serenity`` from ``.port_include.sh``, because ``i686-pc-serenity`` host needs a separate patch

Reply:
But.. we are not a 386bsd system, so what is the significance of 386bsd here?

Reply:
Yeah, config.sub needs to be patched for the correct triplet to work, we should probably keep a fixed config.sub around and simply replace it in any ports that uses autotools, configuring for 386bsd is wrong regardless.